src/collider/GeomTree.cpp

```diff
@@ -133,7 +133,7 @@ GeomTree::GeomTree(const int numVerts, const int numTris, const std::vector<vect
 		PROFILE_SCOPED_DESC("GeomTree::CreateTriTree");
 
 		m_triAABBs.reset(new AABBd[m_numTris]);
-		for (size_t i = 0; i < m_numTris; i++) {
+		for (size_t i = 0; i < (size_t) m_numTris; i++) {
```

Comment:
The other `for` loops in this function use `int i` when iterating over `m_numTris` - should probably follow that precedent instead of a cast here.

In general, why is `m_numTris` (and the other counters in this class) an `int` and not a `size_t`? A really brief look indicates that none of these values can actually be negative..

Reply:
Commit 0551cd44b1 shows what changes when the n_num* members become size_t.

Reply:
LGTM, but I don't know the codebase well-enough to see if this impacts anything else.

Thank you for your effort in helping clean this up!

Reply:
(What I mean is, it requires additional review from a member who is more familiar with the codebase..)

Reply:
Thanks for your review @mwerle!